packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss

```diff
@@ -74,24 +74,25 @@ cbp-tooltip {
   position: relative;
   --cbp-tooltip-placement-gap: var(--cbp-tooltip-gap);
   --cbp-tooltip-placement-offset: var(--cbp-tooltip-offset);
-
+  --cbp-tooltip-gap: var(--cbp-space-4x, 1rem);
+  
   &[open] [role='tooltip']{
       display: flex;
   }
     
-  &:not([definitionlinkstyle="true"]):hover{
+  &:hover{
     color: var(--cbp-tooltip-control-color-hover);
     background: var(--cbp-tooltip-control-color-bg-hover);
   }
 
-  &:not([definitionlinkstyle="true"]):focus-visible,
-  &:not([definitionlinkstyle="true"]):focus-within{
+  &:focus-visible,
+  &:focus-within{
     color: var(--cbp-tooltip-control-color-focus);
     background: var(--cbp-tooltip-control-color-bg-focus);
```

Comment:
where is your outline for the faux-button? I see a white outline in dark mode when it should be black(ish - same color as the icon).